docs/knowledgebase/solution-troubleshooting-etcd-503-service-unavailable-error-peer-discovery.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Solution - Troubleshooting etcd '503 Service Unavailable' Error Message During Peer Discovery"
+linkTitle: "Solution - Troubleshooting etcd '503 Service Unavailable' Error Message During Peer Discovery"
+---
+
+## Issue
+
+You have noticed that Ondat nodes cannot successfully join the cluster. Upon investigation, you also notice the following error messages in after 1 minute of uptime:
```

Comment:
do we specify Ondat nodes or just 'nodes'?

Reply:
fixed